storey/utils.py

```diff
@@ -254,3 +254,12 @@ def find_filters(partitions_time_attributes, start, end, filters, filter_column)
     # for start=1.2.2018 08:53:15, end=5.2.2018 16:24:31, this method will append to filters
     # [(year=2018, month=2,day<=5, filter_column<5.2.2018 16:24:31)]
     _find_filter_helper(partitions_time_attributes, end, "<", "<=", first_uncommon, filters, filter_column)
+
+
+def _drop_reserved_columns(df):
```

Comment:
little bird told me that you are calling this method in mlrun :) so please remove the "_" from the method name